src/screens/lifemap/FamilyMembersNames.js

```diff
@@ -86,7 +86,11 @@ export class FamilyMembersNames extends Component {
 
   addFamilyCount = (text, field) => {
     // if reducing the number of family members remove the rest
-    if (text !== '' && this.getFieldValue('countFamilyMembers') > text) {
+    if (
+      text !== '' &&
+      text !== -1 &&
+      this.getFieldValue('countFamilyMembers') > text
+    ) {
       this.props.removeFamilyMembers(this.draftId, text)
```

Comment:
```suggestion
       this.props.removeFamilyMembers(this.draftId, text === -1 ? 1 : text)
```

You could have the above here and it will do the same as the else if below. It will remove the family members and errors.